src/meta/meta_table.c

```diff
@@ -37,10 +37,23 @@ __metadata_turtle(const char *key)
 int
 __wt_metadata_open(WT_SESSION_IMPL *session)
 {
+	int tret;
```

Comment:
Prefer `WT_DECL_RET` here, and a `ret` variable.  We only use `tret` in complicated situations where there may already be a non-zero `ret`.